Packs/MicrosoftGraphIdentityandAccess/Integrations/MicrosoftGraphIdentityandAccess/MicrosoftGraphIdentityandAccess.py

```diff
@@ -255,6 +330,162 @@ def remove_member_from_role(client: Client, args: dict) -> str:
     return f"User ID {user_id} has been removed from role {role_object_id}"
 
 
+def ip_named_location_get(ms_client: Client, args: dict) -> CommandResults:
+    if ip_id := args.get('ip_id'):
```

Comment:
Redundant, If the argument is marked as required in the integration YML, It can't be empty.